project/Dependencies.scala

```diff
@@ -82,7 +82,8 @@ object Dependencies {
   )
 
   lazy val securityDeps = Seq(
-    "org.apache.shiro" % "shiro-core" % shiro
+    "org.apache.shiro" % "shiro-core" % shiro,
+    "commons-beanutils" % "commons-beanutils" % "1.9.4" exclude("commons-beanutils","commons-beanutils")
```

Comment:
Question: may I know what we need to exclude `commons-beanutils`?

Reply:
Hi Justin,
If not excluding other common-beanutil version,It is downloading all other previous versions and build was getting failed with 
the exception related to beanutil(http://jenkins2.datastax.lan:8080/view/All/job/DSE_Spark_Jobserver_2/235/console).
Including 1.9.4 version and discarding lower version resolved that error and build was successful.

Reply:
Could you please show the dependency tree and which libs are using `common-beanutils`?